osu.Game/Screens/IOsuScreen.cs

```diff
@@ -60,8 +60,9 @@ public interface IOsuScreen : IScreen
 
         /// <summary>
         /// Whether mod track adjustments are allowed to be applied.
+        /// Null means to inherit from the parent screen.
```

Comment:
not sure on this wording, maybe the following will be clearer?

```suggestion
        /// A <see langword="null"/> value means that the parent screen's value of this setting will be used.
```